Source/WebKit/Shared/API/APIObject.cpp

```diff
@@ -33,9 +33,32 @@
 
 #include "WebKit2Initialize.h"
 
+#include <wtf/ThreadSpecific.h>
+
+#define DELEGATE_REF_COUNTING_TO_COCOA PLATFORM(COCOA)
+
 namespace API {
 
+#if DELEGATE_REF_COUNTING_TO_COCOA
+
+WTF::ThreadSpecific<CFTypeRef>& Object::wrapperUnderConstruction()
+{
+    static WTF::ThreadSpecific<CFTypeRef>* result;
```

Comment:
Shouldn't need `WTF::` here and elsewhere in this PR.

Reply:
Also, it would be nice to use `LazyNeverDestroyed` to avoid naked raw pointers like this in new code.